util/find-doc-nits

```diff
@@ -543,6 +563,9 @@ if ( $opt_n ) {
     foreach (@ARGV ? @ARGV : glob('doc/*/*.pod')) {
         &check($_);
     }
+    foreach (@ARGV ? @ARGV : glob('doc/man3/*.pod')) {
+        &check_example_location($_);
+    }
```

Comment:
Instead of looping over man3 PODs a second time, wouldn't it be better to have this inside `check`?

``` perl
    &check_example_location($id, $filename, $contents)
        if $filename =~ m|man3/|;
```

And then, `check_example_location` doesn't need to read the file.

Reply:
Good idea, a very native usage of perl ;-)

Reply:
(and regular expressions)

Reply:
Hmmm, it seems doc/internal/man3 should also be included - which is not done in previous code...

Reply:
Regarding internal documents, I think you should look at the code for `$opt_n` does once more :wink: